circle.yml

```diff
@@ -20,6 +20,9 @@ dependencies:
     - docker build -t oesteban/fmriprep .
 test:
   override:
+    # run unit tests
+    - pwd
+    - python -m unittest discover test
```

Comment:
thank you! I had not looked into this very much but I think you just saved me a lot of time